Clarify additional MinIO Console options available using mc (#1399)
minio/docs: source/administration/minio-console.rst

--------

You can use the MinIO Console for administration tasks like Identity and Access Management, Metrics and Log Monitoring, or Server Configuration.

> It provides access to many common options and complements the more comprehensive command line configurations available using :mc:`mc`.

Reviewer: If we do want to make this stronger:

```suggestion
For more comprehensive or advanced configuration of MinIO, use the :mc:`mc` command line tool.
```

Reviewer: Or to keep more of what you have:

```suggestion
It provides access to  basic configuration settings compared with the comprehensive configurations available in :mc:`mc`.
```
